src/coredump/coredumpctl.c

```diff
@@ -937,13 +936,7 @@ static int run_gdb(sd_journal *j) {
                 _exit(EXIT_FAILURE);
         }
 
-        r = wait_for_terminate(pid, &st);
-        if (r < 0) {
-                log_error_errno(r, "Failed to wait for gdb: %m");
-                goto finish;
-        }
-
-        r = st.si_code == CLD_EXITED ? st.si_status : 255;
+        r = wait_for_terminate_and_check("(gdb)", pid, WAIT_LOG);
```

Comment:
my thinking there was this:

1. all children we are forking off that will eventually become something else through execve() should be named early on after the executable that is going to be executed, but enclosed in `()` to clarify that this isn't the actual binary yet.
2. when waiting for a process we forkedd off and logging about it we should use the process name we forked it off under...

And because of that, I used "(gdb)" as the name in the message here...